### PR TITLE
Cleaning up templates in experimental category

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

**Comment by soninaren** [comment]
fixed #117

File: Templates/QueueTrigger-Python/function.json [hunk 1]
```diff
@@ -15,13 +15,7 @@
...
             "partitionKey": "samples",
             "take": 5,
-            "filter": "status eq '0'"
-        },
-        {
-            "type": "blob",
-            "name": "output",
-            "direction": "out",
-            "path": "samples-output/{id}"
+            "filter": "true"
```
Comment (mathewc) [inline Templates/QueueTrigger-Python/function.json]: Recommend removing this noop filter

File: Templates/QueueTrigger-Powershell/queueTrigger.ps1 [hunk 2]
```diff
@@ -1,8 +1,6 @@
 $in = Get-Content $Env:input
+$output = $Env:output
 
 [Console]::WriteLine("Powershell script processed queue message '$in'")
-
-$output = $Env:output
-$json = $in | ConvertFrom-Json
-$entity = [string]::Format('{{ "timestamp": "{0}", "status": 0, "title": "Powershell Table Entity for message {1}" }}', $(get-date -f MM-dd-yyyy_HH_mm_ss), $json.id)
+$entity = [string]::Format('{{ "timestamp": "{0}", "title": "queue message:{1}" }}', $(get-date).ToString(), $in)
```
Comment (mathewc) [inline Templates/QueueTrigger-Powershell/queueTrigger.ps1]: Missing space in "queue message:{1}"

**Comment by mathewc** [comment]
These changes don't really seem to address #117. The idea in that issue was to make them simple, so they're not a mashup of various bindings. E.g., for these, maybe we just have them get the queue message and that is it.

What do others think? I think we should rely on doc/samples to show the possibilities for various languages, not the template bindings.

**Comment by soninaren** [comment]
The reason i did not remove table binding from these templates is because these are the only 2 templates with table bindings. I feel we should have at least one sample of table input and output binding until we add more.

**Comment by mathewc** [comment]
Ok.

**Comment by soninaren** [comment]
Addressed the PR comments

**Pull request merged**